build/vega-lite-schema.json

```diff
@@ -6071,7 +6059,63 @@
       ]
     },
     "LatLongFieldDef": {
-      "$ref": "#/definitions/FieldDef<\"quantitative\">"
+      "additionalProperties": false,
+      "properties": {
+        "aggregate": {
+          "$ref": "#/definitions/Aggregate",
+          "description": "Aggregation function for the field\n(e.g., `mean`, `sum`, `median`, `min`, `max`, `count`).\n\n__Default value:__ `undefined` (None)"
+        },
+        "bin": {
```

Comment:
Bin should support `null` only here since I declare

```ts
export type LatLongFieldDef<F extends Field> = FieldDefBase<F> &
  TitleMixins &
  Partial<TypeMixins<'quantitative'>> &
  GenericBinMixins<null>;
```

Reply:
https://github.com/vega/ts-json-schema-generator/issues/58

Reply:
This is the case now.